src/ik.rs

```diff
@@ -74,12 +74,21 @@ pub struct Constraints {
     pub rotation_y: bool,
     #[cfg_attr(feature = "serde-serialize", serde(default = "default_true"))]
     pub rotation_z: bool,
+    #[cfg_attr(
+        feature = "serde-serialize",
+        serde(default = "default_empty_string_vector")
```

Comment:
```suggestion
        serde(default)
```

Reply:
Thank you for reviewing. Could you tell me why it is unnecessary?

Reply:
> Could you tell me why it is unnecessary?

- If no function is specified, [serde(default) will call `Default::default()`](https://serde.rs/field-attrs.html#default).
- `Default::default()` of `Vec` returns an empty vector. This is the same as what `default_empty_string_vector` returns.

Reply:
I understand and accept your suggestion . Thank you so much!